packages/path-provider/android/src/main/java/io/flutter/plugins/path_provider/PathProviderPlugin.java

```diff
@@ -46,4 +51,8 @@ private String getPathProviderTemporaryDirectory() {
   private String getPathProviderApplicationDocumentsDirectory() {
     return PathUtils.getDataDirectory(activity);
   }
+
+  private String getPathProviderStorageDirectory() {
+    return Environment.getExternalStorageDirectory().getAbsolutePath();
```

Comment:
The use case I had in mind for this was to get the root of the available file system, such that a file browser could be created within an app. I wanted to create a toy audio book app that would allow the user to navigate to a directory and mark it as the location all audio books could be found under. The latter API call you bring up is not quite targeted for this use case, as it expects you to specify what type of media you are interested in (i.e. for finding the location to save pictures). In my case the user may have located their audio books at any place accessible on the external storage.